### PR TITLE
feat(angular/sidebar): implement dark mode

### PR DESCRIPTION
**Pull request by mhaertwig**

### CODE REVIEW

**Review by matthiaswattingersbb — APPROVED** [review]

File: src/angular/styles/_typography.scss [hunk 1]
```diff
@@ -112,6 +112,9 @@
   --sbb-color-text: var(--sbb-color-black);
   --sbb-color-weak-accent: var(--sbb-color-granite);
 
+  // Dark mode colors
+  --sbb-dark-mode-color-red: #{sbb.$sbbDarkModeColorRed};
```
Comment (jeripeierSBB) [inline src/angular/styles/_typography.scss:116]: In my opinion it sounds better to have `dark-mode` as suffix. What do you think?

Comment (mhaertwig) [reply src/angular/styles/_typography.scss:116]: I actually had this at the beginning but for me it sounded like it was a dark red.  That's why I decided to prefix the colors with `dark-mode`. But am not committed to that. Maybe ask the others in the train what they think 😉

Comment (jeripeierSBB) [reply src/angular/styles/_typography.scss:116]: @kyubisation

Comment (kyubisation) [reply src/angular/styles/_typography.scss:116]: Good question.
Comparing with lean, we did not declare changed variables on their own and just override existing ones.
As another option, we could use a different separator for this, e.g. `--sbb-color-red_dark-mode`.
What do you guys think?

Comment (mhaertwig) [reply src/angular/styles/_typography.scss:116]: We'll need the dark mode red in different places which is why I decided to declare a variable.
Let's go with `--sbb-color-red_dark-mode`.

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Nice work 😃

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**